### ISSUE TITLE
Review / tighten up text

### ISSUE THREAD

**Issue by CenterNancy** [sethjohnson1/collections#62]
Nancy review text of the following with view to make it all more succinct if possible: "Advanced search tips," "My Exhibit" popup with instructions, "About/Help" text

**Comment by CenterNancy** [comment]
How about this for the "About/Help" page?

<h2>Online Collections of the Buffalo Bill Center of the West</h2>

<p>Welcome to our Online Collections! The Center's museums care for thousands of objects—from the common-place to the extraordinary.</p>

<p>In Online Collections, you'll find objects from each museum, many on display but some currently in storage. For some we have a lot of information, for others we don't. Do you know something about an object that we don't, or have a story to share? Please comment or ask questions!</p>

Explore, search, even create your own <a title="My Exhibit" href="/oc/treasures/pack/"><strong>My Exhibit</strong></a>, and view <a title="Virtual Exhibits" href="/oc/usergals/">those made by others</a>. 

<h3>Searching the Collection</h3>

<ul>
	<li>Click on <a title="New Search" href="/oc/"><strong>New Search</strong></a>.</li>
	<li>Use the <strong>Search the Collection</strong> box for keywords.</li>
	<li>Search all museums or narrow your search with the museum checkboxes.</li>
	<li>You can search for what's <strong>On Display</strong> too.</li>
	<li>Use <a title="Advanced Search" href="/oc/treasures/advancedsearch/"><strong>Advanced Search</strong></a> for more control.</li>
	<li>Use <a title="Site Search" href="/oc/treasures/google_search_page/"><strong>Site Search</strong></a> to search our Online Collections site using Google.</li>
</ul>

<h3>Creating a Virtual Exhibit</h3>

<ul>
	<li>Click on <a title="My Exhibit" href="/oc/treasures/pack/"><strong>My Exhibit</strong></a>.</li>
	<li>Search for what you want in your exhibit.</li>
	<li>Hover over a thumbnail and click on the + sign that appears to add the object to our exhibit.</li>
	<li>Or click a thumbnail to view the full record and then click "Add to Virtual Exhibit."</li>
	<li>Click on <a title="My Exhibit" href="/oc/treasures/pack/"><strong>My Exhibit</strong></a> to give it a title and description, and fill in the <strong>Final details</strong>.</li>
	<li>Click <strong>Save exhibit</strong>.</li>
	<li>Check your e-mail for a "Thanks for curating a Virtual Exhibit" message with an edit code to keep&mdash;in case you want to edit later!</li>
</ul>

**Comment by CenterNancy** [comment]
So the instruction pop-up on My Exhibit has to go, right? It's making me homicidal on mobile/ipad...

**Comment by CenterNancy** [comment]
New text for INSTRUCTION POPUP (or whatever form it will take) for My Exhibit page

<h2>Create your own exhibit! Here’s how:</h2>

<ul>
                <li>Browse or search the collection</li>
                <li>Add objects to your new exhibit with the + symbol on thumbnails or the <strong>Add to 

Virtual Exhibit</strong> link on an object's page.</li>
                <li>When you’re done, click on <a title="My Exhibit" href="/oc/treasures/pack/"><strong>My 

Exhibit</strong></a>.</li>
                <li>Add a title, description, and <strong>final details</strong>.
                <li>Click <strong>Save exhibit</strong>.
</ul>

**Comment by sethjohnson1** [comment]
Thanks for doing this! The page looks better already. Did the top part, couple questions:
1. create your own "My Exhibit" or "Virtual Exhibit" 
1. Search the Collection - no 's' on the end?
1. Not sure about the first part of Creating an Exhibit "Click on My Exhibit" - I think we could just start with bullet point 2
1. Should I add the animated GIF and screenshot below the instructions?

I will be working on the popup a little later - it will be mobile-friendly! I am not going to be in tomorrow but I will try to get my weekend changes pulled into test environment. Don't worry about committing changes at the moment, we'll power through this massive revision and get that figured out later.

**Comment by CenterNancy** [comment]
Oops - yes, should be Create your own Virtual Exhibit
Yes on "Search the Collection"
Yes I agree we can eliminate bullet point 1 on Creating a Virtual Exhibit
Yes on adding animated gif and screenshot

Nancy

On May 25, 2015, at 12:58 PM, sethjohnson1 <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for doing this! The page looks better already. Did the top part, couple questions:
1. create your own "My Exhibit" or "Virtual Exhibit"
1. Search the Collection - no 's' on the end?
1. Not sure about the first part of Creating an Exhibit "Click on My Exhibit" - I think we could just start with bullet point 2
1. Should I add the animated GIF and screenshot below the instructions?

I will be working on the popup a little later - it will be mobile-friendly! I am not going to be in tomorrow but I will try to get my weekend changes pulled into test environment. Don't worry about committing changes at the moment, we'll power through this massive revision and get that figured out later.

-

**Comment by sethjohnson1** [comment]
great that much is done, I will pull changes sometime soon (battling
Zoomify right now...)

**Comment by CenterNancy** [comment]
How dare Zoomify be so intractable!

Nancy

On May 25, 2015, at 3:29 PM, sethjohnson1 <notifications@github.com<mailto:notifications@github.com>> wrote:

great that much is done, I will pull changes sometime soon (battling
Zoomify right now...)

On Mon, May 25, 2015 at 3:23 PM, CenterNancy <notifications@github.com<mailto:notifications@github.com>>
wrote:

> Oops - yes, should be Create your own Virtual Exhibit
> Yes on "Search the Collection"
> Yes I agree we can eliminate bullet point 1 on Creating a Virtual Exhibit
> Yes on adding animated gif and screenshot
>
> Nancy
>
> On May 25, 2015, at 12:58 PM, sethjohnson1 <notifications@github.com<mailto:notifications@github.com>
> <mailto:notifications@github.com>> wrote:
>
>
> Thanks for doing this! The page looks better already. Did the top part,
> couple questions:
> 1. create your own "My Exhibit" or "Virtual Exhibit"
> 1. Search the Collection - no 's' on the end?
> 1. Not sure about the first part of Creating an Exhibit "Click on My
> Exhibit" - I think we could just start with bullet point 2
> 1. Should I add the animated GIF and screenshot below the instructions?
>
> I will be working on the popup a little later - it will be
> mobile-friendly! I am not going to be in tomorrow but I will try to get my
> weekend changes pulled into test environment. Don't worry about committing
> changes at the moment, we'll power through this massive revision and get
> that figured out later.
>
> -
> Reply to this email directly or view it on GitHub<
> https://github.com/sethjohnson1/collections/issues/62#issuecomment-105289190
> >.
>
> -
> Reply to this email directly or view it on GitHub
> <https://github.com/sethjohnson1/collections/issues/62#issuecomment-105317152>
> .
>

-

**Comment by CenterNancy** [comment]
Whew! I forgot how complicated writing the "Advanced Search" tips in plain English is. However, I discovered that page repeated some things so this rewrite is quite a bit shorter. We might want to test the links once this is put in there, as some seem to throw in a % sign here and there when I tested accessing the link. Here's the text document with my tags (I know, weird maybe but it's good practice for me to write it that way...)

<h2>Advanced Search tips</h2>

<h3>NOTE: You can try Google Custom Search too:</h3>
[Google search box]

<h4>Advanced searching:</h4>

<ul>
	<li>If you need to click <strong>back</strong> to return to this page, it should retain your entries.</li>
	<li>A space separating terms is treated as AND: <strong>silver necklace</strong> finds everything with “silver” AND “necklace”.</li>
	<li>A MINUS SIGN in front of a term omits it: <strong>silver -necklace</strong> find everything silver without the term necklace.</li>
	<li>Terms in quotes are searched as a phrase: <strong>“blossom necklace”</strong> in Synopsis finds records with that exact phrase in the synopsis.</li>
	<li>Putting a MINUS SIGN inside the quotes omits the term or phrase: <strong>gun “-buffalo bill”</strong> returns “gun” records without the term “buffalo bill” in them.</li>
	<li>Use the PERCENT SIGN as a wildcard&mdash;but with caution. Searching with <strong>deer%</strong>, <strong>%deer%</strong>, <strong>%deer%</strong>, and<strong>% deer %</strong> yields four different sets of results!</li>
	<li>Combine results on different fields for more specific results. Try this.</li>
	<li>After you do an Advanced Search and get the results, doing another search on that results page will search just that data set. If those results are not what you’re looking for, simply click <strong>back</strong> and refine your search. For example: your search for <strong>blossom necklace</strong> yields 9 results; if you now search for <strong>turquoise</strong> within those results, it narrows it down to 7 results.</li>
</ul>

<h4>Fields to use sparingly in a search (and why):</h4>

<ul>
	<li>Date: Our records don’t currently have standardized data in date fields. It could be a date range, an exact date, or maybe a "circa" date like c. 1890, ca. 1890, or even 1890s. So returning good results is difficult.</li>
	<li>Dimensions: This data is also not very standardized, but try it if you like&mdash;you might get lucky!</li>
	<li>Object Title: fewer than 15 percent of our object records list a title, so don't be surprised by few results if you try it. An exception is the exact title of an artwork.</li>
	<li>Gloss: This field is for any additional information or narrative our staff has written about an object; unfortunately fewer than 5 percent of our records have gloss.</li>
	<li>Inscription: This field only has data if the object has an inscription of some sort on it, and most do not.</li>
	<li>Remarks: This field includes descriptions or label text if a record has that information. About 75 percent of our records do so try it if you like; if you don’t get much for results, try Synopsis instead.</li>
</ul>
